smithy-rules-engine/src/main/resources/software/amazon/smithy/rulesengine/testutil/test-cases/default-values.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.0",
```

Comment:
Why are these rolling back to 1.0?

Reply:
The intention is to launch these as just "1.0". Russel was incrementing them just as part of scripting changes.